plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopPlugin.java

```diff
@@ -27,5 +34,14 @@ public void testGetConnectorFactory()
         assertInstanceOf(getOnlyElement(plugin.getConnectorFactories()), AtopConnectorFactory.class);
     }
 
-    // TODO test factory
+    @Test
+    public void testCreateConnector()
+            throws Exception
+    {
+        Plugin plugin = new AtopPlugin();
+        ConnectorFactory factory = getOnlyElement(plugin.getConnectorFactories());
+
+        Path atopExecutable = Files.createTempFile(null, null);
+        factory.create("test", ImmutableMap.of( "atop.executable-path", atopExecutable.toString()), new TestingConnectorContext());
```

Comment:
Could you call append `.shutdown()` as other `Test*Plugin`?

Reply:
Also, please fix checkstyle failure:
```
Error:  src/test/java/io/trino/plugin/atop/TestAtopPlugin.java:[45,47] (whitespace) ParenPad: '(' is followed by whitespace.
```

Reply:
fixed.